manifest.json

```diff
@@ -400,6 +400,97 @@
               "variant": "v7"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "arm",
+              "dockerfile": "src/debian/9.0/arm32v7",
```

Comment:
It feels like the folder should be `9` instead of `9.0` as that is the base image the Dockerfile is based on - `debian:9`.  `debian.9` looks to currently be 9.7.